ami/main/migrations/0085_project_activity_sort_indexes.py

```
@@ -0,0 +1,34 @@
+    operations = [
+        # Runtime SET overrides the startup "-c statement_timeout" option and
+        # persists for the rest of this (non-atomic) migration's connection.
+        migrations.RunSQL(
+            sql="SET statement_timeout = 0;",
+            reverse_sql=migrations.RunSQL.noop,
+        ),
+        AddIndexConcurrently(
+            model_name="occurrence",
+            index=models.Index(fields=["project", "-updated_at"], name="occur_proj_updated_desc_idx"),
+        ),
+    ]
```

Ruff (0.15.13)

[warning] Mutable default value for class attribute

(RUF012)